clients/client-python/tests/unittests/test_metalake_ut.py

```diff
@@ -0,0 +1,39 @@
+"""
+Copyright 2024 Datastrato Pvt Ltd.
+This software is licensed under the Apache License version 2.
+"""
+
+import unittest
+
+from gravitino.dto.responses.metalake_response import MetalakeResponse
+
+
+class TestMetalakeUT(unittest.TestCase):
```

Comment:
Since we already have two different folders for UT and IT, I think it is not necessary to add "UT" as a prefix, "TestMetalake" should be enough?

Reply:
OK